robot/Cumulus/tests/browser/manage_households/change_naming_managehh.robot

```diff
@@ -1,35 +1,68 @@
-    ${loc}    Validate Checkbox    &{contact2}[FirstName] &{contact2}[LastName]    Formal Greeting
-    Double Click Element    ${loc}
-    Click Button       title=Save
-    Unselect Frame
-    Wait Until Page Contains    Account Owner
-    Sleep    3
-    Go To Object Home          Account
-    Click Link    link=&{contact1}[LastName] and &{contact2}[LastName] Household
-    Select Tab     Details
-    Check Field Value    Informal Greeting    &{contact2}[FirstName]
-    Check Field Value    Formal Greeting    &{contact1}[FirstName] &{contact1}[LastName]
+Change Name Display Settings on Manage Household Page
+    [Documentation]                      Create two contacts using the backend API
+    ...                                  Using Manage Household, add the second contact part of the Contact#1's household
+    ...                                  From the manage household page's name display settings, choose the options for both the contacts and save
+    ...                                  Verify the selection made is retained and the right display is shown on the Details section
+    [tags]                               W-038348             feature: Manage Households
+
+    Go To Page                              Details
+    ...                                     Account
+    ...                                     object_id=&{contact1}[AccountId]
+    Click Link                              link=Manage Household
+    Go To Page                              Custom                                                              ManageHousehold
+    Add contact                             Existing                                                            &{contact2}[FirstName] &{contact2}[LastName]
+    Current Page Should Be                  Details                                                             Account
+    Wait Until Loading Is Complete
+    Click Link                              link=Manage Household
+    Go To Page                              Custom                                                              ManageHousehold
+
+    # Choose the display option as Informal Greeting for contact#1
+
+    Validate And Select Checkbox            &{contact1}[FirstName] &{contact1}[LastName]
+    ...                                     Informal Greeting
+
+    # Choose the display option as Formal Greeting for contact#2
+
+    Validate And Select Checkbox            &{contact2}[FirstName] &{contact2}[LastName]
+    ...                                     Formal Greeting
+
+    Save Changes Made For Manage Household
+    Current Page Should Be                  Details                                                             Account
+    Wait Until Page Contains                Account Owner
+
+    Go To Page                              Listing                                                             Account
+
+    Click Link                              link=&{contact1}[LastName] and &{contact2}[LastName] Household
```

Comment:
use go to page Details Account here too to avoid some clicks

Reply:
done